modules/kubernetes-addons/csi-secrets-store-provider-aws/README.md

```diff
@@ -10,16 +10,20 @@ AWS Secrets Manager and Config Provider for Secret Store CSI Driver allows you t
 
 See the [csi-secrets-store-provider-aws](https://github.com/aws/eks-charts/tree/master/stable/csi-secrets-store-provider-aws).
 
-<!--- BEGIN_TF_DOCS --->
```

Comment:
Outdated terraform-docs flag so docs were out of sync (see updates directly below)